spec/views/idv/shared/_document_capture.html.erb_spec.rb

```diff
@@ -105,7 +105,7 @@
     it 'sends selfie_capture_enabled to the frontend' do
       render_partial
       expect(rendered).to have_css(
-        "#document-capture-form[data-doc-auth-selfie-capture='false']",
+        "#document-capture-form[data-doc-auth-selfie-capture='true']",
```

Comment:
Can someone who knows this code better than I confirm this was the right thing to do?

Reply:
This looks like what I would expect. This looks duplicative of the test below on 119 now though. I think we could remove one for the same effect.